docs/application/dotnet/guides/nlp/overview.md

```diff
@@ -0,0 +1,13 @@
+# Natural Language Processing
+
+The natural language processing features introduce how you can process sentence with Nlp class. They also describe how you can call methods about Nlp in your application.
+
+You can use the following natural language processing features in your DOTNET applications:
+
+-   [Natural Language Processing](nlp.md)
+
+    You can use natural language processing functionalities in your application, such as language detect, word_tokenize, ne_chunk, and pos_tag. Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processin.
```

Comment:
Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (Nlp) functionalities, such as language detection, parts of speech, word toeknize, and entity detection.